src/core/builtins/builtins_http.ml

```diff
@@ -105,7 +105,10 @@ let add_http_request ~stream_body ~descr ~request name =
       let headers =
         List.map (fun (x, y) -> (Lang.to_string x, Lang.to_string y)) headers
       in
-      let timeout = Lang.to_int (List.assoc "timeout_ms" p) in
+      let timeout =
+        List.assoc "timeout_ms" p |> Lang.to_option |> Option.map Lang.to_int
+        |> Option.value ~default:10000
```

Comment:
```suggestion
        Option.value ~default:1000 (Lang.to_valued_option Lang.to_int (List.assoc "timeout_ms" p))
```

Reply:
Actually this is not good. We want `null()` for timeout to mean "no timeout set". Will fix.